dataflux_pytorch/dataflux_iterable_dataset.py

```diff
@@ -90,6 +92,22 @@ def data_format_default(data):
     return data
 
 
+def _get_missing_permissions(storage_client: any, bucket_name: str,
```

Comment:
This is probably fine, but it feels a little weird to import `dataflux_iterable_dataset` from `dataflux_mapstyle_dataset`. It might make sense to put it into its own file. I think other Python projects often create a file with a leading underscore in the name in order to signify that it's internal and not something that users should import.

Reply:
Created a helper file for this.